webpack.config.js

```diff
@@ -55,6 +56,9 @@ const config = {
   plugins: [
     new MiniCssExtractPlugin(),
     new VueLoaderPlugin(),
+    new DefinePlugin({
```

Comment:
I do not understand what is achieved by DefinePlugin here.

Reply:
Me neither. I hoped You got an Idea. It seems to fix the issue, but why - I can't say. :-/